vc

```diff
@@ -136,6 +136,10 @@ set +e
 	if [ -n "$message" ]; then
 		echo -e "- $message"
 	elif [ -n "$content" ]; then
+	        cat "$content" > format_tmp
+                sed -i '1s;^;- ;' format_tmp
+                cat format_tmp
+                rm format_tmp
```

Comment:
these 4 lines can be replaced with

sed -i '1s;^;- ;' "$content"

no?

Reply:
No. Because this way the file the content comes from gets modified. And on a second run there are two dashes. 

And I don't want to modify the files the user gives as source.